### PR TITLE
MC-13931 Add documentation for creating a serverless script.

### PR DESCRIPTION
**Pull request by ehudon**
#### Changes made

- Add the document for creating serverless scripts.
- Fixed URLs in the example that were not valid. See #321
- Fixed the documentation for the code being returned by the API. It's not in returned in base64. #321
- Changed the siteId in the examples across all operations so we are consistent within the same document.

#### Related PRs
- https://github.com/cloudops/cloudmc-stackpath-plugin/pull/200
- https://github.com/cloudops/cloudmc-stackpath-plugin/pull/197
- https://github.com/cloudops/cloudmc-api-docs/pull/321

### CODE REVIEW

File: source/includes/stackpath/_scripts.md [hunk 1]
```diff
@@ -109,18 +131,92 @@ Attributes | &nbsp;
...
-`code`<br/>*string* | Base64 encoded contents of a script.
+`code`<br/>*string* | The contents of the script.
 `version`<br/>*string* | The version number of the script.
 `routes`<br/>*Array[string]* | The routes that incoming requests should respond with a script.
 
+<!-------------------- CREATE A SCRIPT -------------------->
+
+### Create a script
+
+Create a new serverless script.
```
Comment (razine-bensari) [inline source/includes/stackpath/_scripts.md]: A good addition would be to mention where and when it is executed

Comment (razine-bensari) [reply source/includes/stackpath/_scripts.md]: Assuming it was not mentioned in another documentation. Maybe it is mentioned in the Retrieve script docs. Ill check

**Review by razine-bensari — APPROVED** [review]
🔥

File: source/includes/stackpath/_scripts.md (hunk 1 above)
Comment (razine-bensari) [reply source/includes/stackpath/_scripts.md]: Nevermind already taken care of!

**Review by aaronalmeida — APPROVED** [review]
Thanks for fixing the errors! LGTM 🚀

**Comment by ehudon** [comment]
@razine-bensari I implemented your suggestions. Please let me if you are site good with the PR.

**Review by razine-bensari — APPROVED** [review]
💯  thks for taking care of the improvment @ehudon

**Pull request merged**